dsc/TOC.md

```diff
@@ -54,6 +54,10 @@
 ## [Using a DSC report server](reportServer.md)
 ## [Pull server best practices](secureServer.md)
 
+# [DSC examples](dscExamples.md)
+## [Building a CI-CD pipeline with DSC, Pester, and Visual Studio Team Services](dscCiCd.md)
```

Comment:
I'm open to expanding the acronyms. I'd at least like to keep DSC in the title for SEO reasons, and so people understand that this is DSC-related when they hit it. There are a lot of topics on CI/CD out there.